app/services/aptrust/aptrust_uploader.rb

```diff
@@ -368,10 +368,16 @@ def bag_export
     status = export_data
     if status == ::Aptrust::EVENT_EXPORTED
       bag_manifest
+      msg_handler.msg_error "bag.complete? return false" unless bag.complete?
+      msg_handler.msg_error "bag.consistent? return false" unless bag.consistent?
+      msg_handler.bold_debug [ msg_handler.here, msg_handler.called_from,
+                               "bag.complete?=#{bag.complete?}",
+                               "bag.consistent?=#{bag.consistent?}",
+                               "" ] if debug_verbose
       track( status: ::Aptrust::EVENT_BAGGED, note: "bag_dir: #{bag_dir}" )
       status = ::Aptrust::EVENT_BAGGED
     end
-    bag.write_bag_info( info ) # Update/create bagit-info.txt file
+    # bag.write_bag_info( info ) # this causes it generate an incorrect bag checksum
```

Comment:
@fritzfreiheit, did you try removing line 366 and keep this line you're commenting out? That'd be more similar to what I'm doing, FWIW:
https://github.com/mlibrary/bag-courier/blob/2fa9a5ee2701eb3b193b041618163ab4ee8dfb0a/lib/bag_courier.rb#L119-L135

Also, I'm curious why things started failing more recently, if you were using this ordering for a while now...

Reply:
@ssciolla -- Yeah, I tried it the way you suggested, and it failed. Interestingly, it worked with both lines from testing and localhost.